service/collector/collector.go

```diff
@@ -21,17 +21,19 @@ type Config struct {
 	K8sClient kubernetes.Interface
 	Logger    micrologger.Logger
 
-	AwsConfig        awsutil.Config
-	InstallationName string
+	AwsConfig               awsutil.Config
+	InstallationName        string
+	TrustedAdvisorSupported bool
```

Comment:
Why is it `HasTrustedAdvisorSupportPlan` and `TrustedAdvisorSupported`? I think it would be good to have one flag. I would also like to have more structure here. Something like `.Values.Installation.V1.Provider.AWS.TrustedAdvisor.Enabled` would make me happy. Just thinking loud though.

Reply:
My thinking was that `HasTrustedAdvisorSupportPlan` is kind of obvious and self-explaining for configuration file, but overly verbose for code. `.Values.Installation.V1.Provider.AWS.TrustedAdvisor.Enabled` works for me as well.

Reply:
IMO it creates more complexity to change the name. Having one identifier makes it way easier to work with. Understanding, maintaining, and refactoring wise.